Initial support for Hailo-8L (#12431)
blakeblackshear/frigate: docs/docs/configuration/object_detectors.md

- All models are automatically downloaded and stored in the folder `config/model_cache/rknn_cache`. After upgrading Frigate, you should remove older models to free up space.
- You can also provide your own `.rknn` model. You should not save your own models in the `rknn_cache` folder, store them directly in the `model_cache` folder or another subfolder. To convert a model to `.rknn` format see the `rknn-toolkit2` (requires a x86 machine). Note, that there is only post-processing for the supported models.

## Hailo-8l

> This detector is availability if you are using the Raspberry pi 5 with ai kit and currently not tested if you are using the Hailo-8L with other hardware.

Reviewer: ```suggestion
This detector is available if you are using the Raspberry Pi 5 with Hailo-8L AI Kit. This has not been tested using the Hailo-8L with other hardware.
```

Author: Thank you again. first pull request to a big repo.
